src/screens/OtherProfile/index.js

```diff
@@ -636,7 +644,7 @@ const styles = StyleSheet.create({
     fontWeight: 'bold',
     fontSize: 18,
     lineHeight: 22,
-    color: colors.black,
+    color: COLORS.black,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated to use `COLORS.black` from the `theme` module. This is a good practice for maintaining consistency across the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->